CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 # Release Notes
 
-## Platform Version 0.10.9 - UNRELEASED
+## Platform Version 0.10.9 - 2023-05-11
+
+### Added
+
+* Simplified control plane api ([#3073](https://github.com/infinyon/fluvio/issues/3073))
+* Add helper method to get utf8 from data ([#3236](https://github.com/infinyon/fluvio/issues/3236))
+* Move x records outputed from smdk test to --verbose #3169 ([#3234](https://github.com/infinyon/fluvio/issues/3234))
+* Add message about record too large on produce cli ([#3242](https://github.com/infinyon/fluvio/issues/3242))
+
+### CI
+
+* `cdk build` uses a default target ([#3222](https://github.com/infinyon/fluvio/issues/3222))
```

Comment:
update to include last commit: `cdk test` and `cdk deploy` also uses default target

Reply:
updated